### PR TITLE
feat: Update header and PageLayout components

### PR DESCRIPTION
**Pull request by MilosMarkoni**

### CODE REVIEW

File: packages/ui/src/styles/colors.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const Colors = {
   light: {
     headerBackground: '#F6F8FA',
     logoBackground: '#57606A',
-    background: '#FFFFFF',
+    background: '#012749',
```
Comment (MilosMarkoni) [inline packages/ui/src/styles/colors.ts]: Ignore these lines, will be updated

File: packages/ui/src/Layouts/PageLayoutHeader.tsx [hunk 2]
```diff
@@ -96,3 +96,11 @@ const HeaderLinksWrapper = styled.div`
   justify-content: center;
   display: flex;
 `
+const HeaderButtonsWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  & button {
```
Comment (aleksaToljic) [inline packages/ui/src/Layouts/PageLayoutHeader.tsx]: Nitpick, no need for `&` because we also have empty space after it

**Comment by aleksaToljic** [comment]
I switched PR status to ready-for-review, so that Chromatic would be triggered

File: packages/ui/src/Layouts/PageLayoutHeader.tsx (hunk 2 above)
Comment (MilosMarkoni) [reply packages/ui/src/Layouts/PageLayoutHeader.tsx]: updated :)

**Review by aleksaToljic — APPROVED** [review]

**Pull request merged**